src/order-history/OrderHistoryPage.jsx

```diff
@@ -70,11 +70,10 @@ class OrderHistoryPage extends React.Component {
 
   renderLineItems(lineItems) {
     return lineItems.map(({
-      itemId,
       description,
       quantity,
     }) => (
-      <p className="d-flex" key={itemId}>
+      <p className="d-flex" key={description}>
```

Comment:
Do we know that `description` will be unique?  Is it safer to add a second argument `index` to map and use that?

Reply:
I this case I think the description is unique. We could use the index as it turns out: https://reactjs.org/docs/lists-and-keys.html#keys Though I do know our linter will yell at us.